test/e2e/tests/test_ca_activation.py

```diff
@@ -91,6 +92,10 @@ def simple_certificate_authority():
 
     yield (ca_cr,ca_ref, ca_name)
 
+    #Disable CA if status is ACTIVE
+    acmpca_validator = ACMPCAValidator(acmpca_client)
+    acmpca_validator.disable_active_ca(ca_resource_arn)
```

Comment:
Why would we need to disable the CA manually?

Reply:
If the test fails and the CA is left as ACTIVE, I added this line to disable the CA before it gets deleted.